packages/discovery-styles/scss/components/document-preview/_document-preview-pdf-viewer.scss

```diff
@@ -46,9 +46,14 @@
 }
 
 .#{$prefix}--document-preview-pdf-viewer-highlight__item {
+  box-sizing: content-box;
+  margin: -3px;
+  padding: 1px 1px 0;
   position: absolute;
   opacity: 20%;
-  background: darken(saturate($highlight, 50%), 40%);
+  background: purple;
+  // background: darken(saturate($highlight, 50%), 40%);
```

Comment:
What's going on here with background? I don't see `purple` getting rendered when I run components linked to tooling. Your commented out line suggests you were experimenting with adjusting the darkness of a background color. Did you opt out of that plan, or is that something you're still playing with?

Reply:
Yes, was for testing. Now restored.